playbooks/researchdata.yml

```
@@ -17,5 +19,5 @@
     - name: tell everyone on slack you ran an ansible playbook
       slack:
         token: "{{ vault_pul_slack_token }}"
-        msg: "{{ inventory_hostname }} completed"
+        msg: Ansible ran {{ ansible_play_name }} on {{ inventory_hostname }}
```

Oh, I forgot! Those weren't quotation marks, they were backticks to make the `play name` stand out in Slack! The inventory_hostname stands out because slack interprets it as a link, but the `play name` needs the backticks. Sorry I took them out before - can we reinstate them?
```suggestion
        msg: Ansible ran `{{ ansible_play_name }}` on {{ inventory_hostname }}
```